client/containers/Pub/PubDocument/PubInlineSuggestedEdits.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react';
+
+import { buttons, FormattingBarSuggestedEdits } from 'components/FormattingBar';
+import { acceptSuggestedEdits } from 'client/components/Editor/plugins/suggestedEdits/resolve';
+
+import { usePubContext } from '../pubHooks';
+
+require('./pubInlineSuggestionMenu.scss');
+
+const shouldOpenBelowSelection = () => {
+	return ['Android', 'iPad', 'iPhone'].some((device) =>
+		navigator.userAgent.toLowerCase().includes(device.toLowerCase()),
+	);
+};
+
+const PubInlineSuggestedEdits = () => {
+	const { collabData, pubBodyState } = usePubContext();
+	const { editorChangeObject } = collabData;
+
+	const selection = collabData.editorChangeObject!.selection;
+
+	const shouldHide = useMemo(() => {
+		const selectionInSuggestionRange = (): boolean => {
+			if (!collabData.editorChangeObject || !collabData.editorChangeObject.view) return false;
+			const state = collabData.editorChangeObject.view.state;
+			return acceptSuggestedEdits(state);
+		};
+
+		const inRange = selectionInSuggestionRange();
+
+		return !inRange || !selection;
+	}, [collabData.editorChangeObject, selection]);
+
+	// range of editable editor space
```

Comment:
This isn't really true, right? This is the bounding box for the selection itself, not the whole editor.